### PR TITLE
[DOCS]chore: refactor header

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      '&::after': {
+        transform: 'translate(1px, 1px)  rotate(45deg)!important',
+      },
+    },
+  },
+  'l1-false': {},
+  'l2-true': {
+    selectors: {
+      '&::before': {
+        transform: `translate(${vars.sizes.$2}, ${vars.sizes.$1}) rotate(135deg)!important`,
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts:109]: this variant needs to overwrite the other one.
and it seems that the order will not work. It needs an important

File: packages/apps/docs/src/components/Layout/components/Header/styles.css.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    borderRadius: '$lg',
+    width: '$11',
+    height: '$11',
+    cursor: 'pointer',
+    color: '$neutral2',
+  }),
+
+  {
+    border: 0,
+    transition: `opacity 0.2s ease 0s`,
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/Header/styles.css.ts:20]:
```suggestion
    transition: `opacity 0.2s ease`,
```

File: packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+});
+
+export const treeListRootVariantClass = styleVariants({
+  isRoot: {
+    height: 'auto!important',
+  },
+  isNotRoot: {},
+});
+
+export const styledLinkClass = style([
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts:34]:
```suggestion
export const linkClass = style([
```

File: packages/apps/docs/src/components/Layout/global.css.ts [hunk 4]
```diff
@@ -13,5 +16,14 @@ export const globalClass = style({
     [$$pageWidth]: '1440px', //sizes
     [$$leftSideWidth]: '265px', //sizes
     [$$backgroundOverlayColor]: 'rgba(250,250,250, .8)', //colors
+    [$$borderColor]: 'rgba(112, 121, 123, 0.2)', //colors
+  },
+  selectors: {
+    [`${darkThemeClass} &`]: {
+      vars: {
+        [$$backgroundOverlayColor]: 'rgba(5,5,5, .8)', //colors
+        [$$borderColor]: 'rgba(240, 240, 240, 0.2)', //colors
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/global.css.ts:25]: Don't we have these colors in react-ui package? Is it good to define it in react-ui?

File: packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+      '&::before': {
+        position: 'absolute',
+        left: `calc(0px - ${vars.sizes.$2})`,
+        content: '',
+        width: vars.sizes.$2,
+        height: vars.sizes.$2,
+        borderRight: `2px solid ${$$borderColor}`,
+        borderTop: `2px solid ${$$borderColor}`,
+        transform: `translate(${vars.sizes.$2}, ${vars.sizes.$1}) rotate(45deg)`,
+        transition: 'transform .2s ease ',
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts:167]:
```suggestion
        transition: 'transform .2s ease',
```

File: packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+export const treeItemPseudoVariantClass = styleVariants({
+  l1: {
+    selectors: {
+      '&::after': {
+        position: 'absolute',
+        right: vars.sizes.$1,
+        content: '+',
+        fontWeight: vars.fontWeights.$light,
+        transform: 'translate(0, 0) rotate(0)',
+        transition: 'transform .2s ease ',
```
Comment (realdreamer) [inline packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts:149]:
```suggestion
        transition: 'transform .2s ease',
```

File: packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts (hunk 3 above)
Comment (sstraatemans) [reply packages/apps/docs/src/components/Layout/components/TreeMenu/styles.css.ts:34]: good one

File: packages/apps/docs/src/components/Layout/global.css.ts (hunk 4 above)
Comment (sstraatemans) [reply packages/apps/docs/src/components/Layout/global.css.ts:25]: we dont. the opacity thing is specific for the documentation website atm

**Review by realdreamer — APPROVED** [review]

**Pull request merged**